core/src/main/java/com/amplifyframework/devmenu/EnvironmentInfo.java

```diff
@@ -60,14 +62,14 @@ public EnvironmentInfo() {
      * @return version information for each plugin added
      */
     public String getPluginVersions() {
-        String pluginVersions = getCategoryPluginVersions(Amplify.Analytics) +
-                getCategoryPluginVersions(Amplify.API) +
-                getCategoryPluginVersions(Amplify.Auth) +
-                getCategoryPluginVersions(Amplify.Logging) +
-                getCategoryPluginVersions(Amplify.Storage) +
-                getCategoryPluginVersions(Amplify.Hub) +
-                getCategoryPluginVersions(Amplify.DataStore) +
-                getCategoryPluginVersions(Amplify.Predictions);
+        // TODO: can we centralize these somewhere? We want ~1 file in the code base that lists all the categories.
+        List<Category<?>> categories = Arrays.asList(Amplify.Analytics, Amplify.API, Amplify.Auth, Amplify.Logging,
```

Comment:
I'm trying to think: what would be the _harm_ of making this `CATEGORIES` public? If it were, then you could just do `Amplify.CATEGORIES.values()`, here. We'd want to change [this one to `Immutable.of(...)`](https://github.com/aws-amplify/amplify-android/blob/main/core/src/main/java/com/amplifyframework/core/Amplify.java#L99), so that nobody could inject/modify categories or play games like that, I guess.

Reply:
> I'm trying to think: what would be the harm of making this CATEGORIES public? 

One downside is that `CATEGORIES` would then show up in autocomplete after typing `Amplify.`, even though there's really no utility for it, which could cause confusion for customers.   Not a huge deal, but something to consider.

> We'd want to change [this one to Immutable.of(...)](https://github.com/aws-amplify/amplify-android/blob/main/core/src/main/java/com/amplifyframework/core/Amplify.java#L99)

When doing this, you'd want to make sure the `Immutable` utility class can handle a `LinkedHashMap`.  It would need to maintain the order, and I'm not sure if it would or not.